app/grants/templates/grants/milestones.html

```diff
@@ -0,0 +1,82 @@
+    {% include 'shared/cards.html' %}
+  </head>
+  <body class="interior {{active}} grant">
+    {% include 'shared/tag_manager_2.html' %}
+    <div class="container-fluid header dash">
+      {% include 'shared/nav.html' %}
+    </div>
+
+    <div class="container milestones">
+      <div class="row justify-content-md-center">
+        <div class="col-sm-12 col-md-12 col-lg-8">
+          <div class="form__intro">
+            <h1>Create Milestones for {{ grant.title }}</h1>
+            <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec eros est, aliquam at orci sed, interdum tincidunt dui. Duis vitae malesuada magna. Morbi volutpat augue id commodo finibus.</p>
+          </div>
+          {% if milestones %}
+            <div class="grant__milestones-timeline">            
+              {% for milestone in milestones %}
+                <div class="grant__milestones-timeline-item" timeline-item="{{ milestone.title }}">
+                  {% if milestone.completion_date %}
+                    <h6>{{ milestone.due_date }}, Completed {{ milestone.completion_date }}</h6>
+                  {% else %}
+                    <form method="POST" action="{% url 'grants:milestones' grant.id %}">
+                      {% csrf_token %}
+                      <input type="hidden" name="method" value="">
+                      <input type="hidden" name="milestone_id" value="{{milestone.id}}">
+                      <a data-form-method="DELETE">Delete</a>
+                      <h6>{{ milestone.due_date }}</h6>
+                      <input type="date" class="form__input" placeholder="Date" name="completion_date" required />
+                      <a data-form-method="PUT">Complete</a>
+                    </form>
+                  {% endif %}
+                  <p>{{ milestone.description }}</p>
+                </div>
+              {% endfor %}
+            </div>
+          {% else %}
+            <div class="grant__milestones-timeline">Donec iaculis quis eros id cursus. Interdum et malesuada fames ac ante ipsum primis in faucibus. Curabitur convallis erat et elit volutpat, vel facilisis odio lacinia.</div>
+          {% endif %}
+          <form action="{% url 'grants:milestones' grant.id %}" method="POST">
+            {% csrf_token %}
+            <input type="hidden" name="method" value="POST">
+
+            <div class="form__group-horizontal">
+              <div class="row">
+                <div class="col-12">
+                  <input type="text" class="form__input" placeholder="Title" name="title" required />
+                  <input type="date" class="form__input" placeholder="Date" name="due_date" required />
+                  <textarea class="form__input" type="text" placeholder="Description" name="description" required></textarea>
+                </div>
+              </div>
+            </div>
+
+            <div class="form__footer">
+              <button class="button button--primary">Create Milestone</button>
```

Comment:
Can you wrap the copy throughout in `{% trans "copy" %}` and add `i18n` to the `{% load %}` at the top of the template?